generators/app/questions/services.js

```diff
@@ -7,13 +7,20 @@ module.exports = [{
     'JWT'
   ]
 }, {
+  type: 'input',
+  name: 'services:jwt-secret',
```

Comment:
Better to split it like `services:jwt:secret-key`. Because when another cipher can be integrated, that "another" cipher can have more options. So in this way it will be kinda splitting answers to sections.